frontend/src/components/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -115,14 +117,39 @@ const AdminDashboard = (): React.ReactElement => {
         },
       },
       {
-        name: "published",
-        label: "Published",
+        name: "status",
+        label: "Status",
         options: {
           customFilterListOptions: {
             render: (v) => {
-              return [`Published: ${v}`];
+              return [`Status: ${v}`];
             },
           },
+          customBodyRender: (value) => {
+            const colorScheme = value === "Published" ? "green" : "orange";
+            return <Tag colorScheme={colorScheme}>{value}</Tag>;
+          },
+        },
+      },
+      {
+        name: "actions",
+        label: " ",
+        options: {
+          customBodyRender: () => {
+            return (
+              <div>
+                <Tooltip label="Edit review">
```

Comment:
Do we want these to be Icon buttons instead of just tooltips? These should be clickable to do the actions right?

Reply:
Yeah I had originally left them out of this PR since I thought we would implement the buttons for the individual edit, preview, delete tickets, but I think it makes more sense to add the buttons here and just update the on change later. Will change! :)